packages/toolbar/src/browser/easy-search-toolbar-item.tsx

```diff
@@ -0,0 +1,145 @@
+/********************************************************************************
+ * Copyright (C) 2022 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { CommandContribution, CommandRegistry, CommandService, MenuContribution, MenuModelRegistry } from '@theia/core';
+import { ContextMenuRenderer, quickCommand } from '@theia/core/lib/browser';
+import { inject, injectable, interfaces } from '@theia/core/shared/inversify';
+import * as React from '@theia/core/shared/react';
+import { quickFileOpen } from '@theia/file-search/lib/browser/quick-file-open';
+import { SearchInWorkspaceCommands } from '@theia/search-in-workspace/lib/browser/search-in-workspace-frontend-contribution';
+import { WorkspaceService } from '@theia/workspace/lib/browser';
+import { AbstractMainToolbarContribution } from './abstract-main-toolbar-contribution';
+import { SearchInWorkspaceQuickInputService } from './search-in-workspace-root-quick-input-service';
+import { MainToolbarMenus, ReactInteraction } from './main-toolbar-constants';
+import {
+    ReactTabBarToolbarContribution,
+    ToolbarAlignment,
+} from './main-toolbar-interfaces';
+
+export const SEARCH_FOR_FILE = {
+    id: 'main.toolbar.search.for.file',
+    category: 'Search',
+    label: 'Search for a File',
+};
```

Comment:
Good call, I had originally wrapped that command because the contextmenurenderer was passing an AnchorArg and breaking the command, but I've added the `includeAnchorArg` option which solves the problem and able to remove this command.